docs/source/frequently_asked_questions.rst

```diff
@@ -1,6 +1,8 @@
 FAQs - Frequently asked questions
 =================================
-
+| **Q:** How do I know what order to put the parameter values in the pytorch tensor which gets passed to the simulator?
+| **A:** If you are using the built-in simulator (caustics.Lens_Source), then you can get the parameters using your_simulator.state_dict(). The parameters whose values are set dynamically will say "None", while the static parameters will have their values shown. The order of the dynamical parameters corresponds to the order you should use in your parameter value tensor. 
```

Comment:
I think this works for any simulator object, not just the Lens_source one

Reply:
I agree. Should we just delete the '(caustics.Lens_Source)' example?

Reply:
yup sounds good

Reply:
done